formats/src/main/scala/org/scanamo/DerivedDynamoFormat.scala

```diff
@@ -1,31 +1,29 @@
 package org.scanamo
 
-import cats.data.{NonEmptyList, Validated}
+import cats.data.{Validated, ValidatedNel}
 import cats.syntax.either._
-import com.amazonaws.services.dynamodbv2.model.AttributeValue
 import org.scanamo.error._
 import org.scanamo.export.Exported
 import shapeless._
 import shapeless.labelled._
 
-import collection.JavaConverters._
-
 trait DerivedDynamoFormat {
-  type ValidatedPropertiesError[T] = Validated[InvalidPropertiesError, T]
+  type ValidatedPropertiesError[T] = ValidatedNel[(String, DynamoReadError), T]
```

Comment:
Would it make sense to have a type alias `fieldname` rather than String, just to document what the first element of the tuple actually is?